Part-04(Create-A-Movie-Website)/Lesson_2/Break_Time.py

```diff
@@ -0,0 +1,10 @@
+import time
+import webbrowser
+
+total_breaks = 3
+break_count = 0
+print("This program started on " + time.ctime())
+while break_count < total_breaks:
+    time.sleep(5)
+    webbrowser.open("https://www.youtube.com/watch?v=wBs4x9kjQP8&t=2096s")
+    break_count = break_count + 1
```

Comment:
To do, 
1 - Repeat 3 Times
2 - Wait 2 hours
3 - Open browser

Note, 
1 - Repeat 3 Times
* using while loop

2 - Wait 2 hours
* using time.sleep function
* time.sleep(2) is two seconds and not two hours
* ctime is current time

3 - Open browser
* using webbrowser.open function
* Python is having difficulty, understanding 
what webbrowser is. We can fix this, by saying 
import webbrowser at the top.